### PR TITLE
[FIX] Fix undefined values in CustomGas component

### PR DESCRIPTION
**Pull request by Cal-L**
**Description**

Sentry reported that CustomGas component experienced undefined values that led to crashers
1. https://github.com/MetaMask/metamask-mobile/issues/2762
2. https://github.com/MetaMask/metamask-mobile/issues/2761

**Fix**
Upon inspection, CustomGas component wasn't being used anymore. This PR focuses on removing CustomGas component and related code.

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

Resolves #???
https://github.com/MetaMask/metamask-mobile/issues/2762
https://github.com/MetaMask/metamask-mobile/issues/2761

### CODE REVIEW

**Review by sethkfman — COMMENTED** [review]
Just a minor comment.

File: app/core/Engine.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ import {
 	WalletDevice,
 	GasFeeController,
 	TokensController,
-	CollectiblesController
+	CollectiblesController,
```
Comment (sethkfman) [inline app/core/Engine.js:23]: I don't think we want to convert this file back to `.js`

Comment (Cal-L) [reply app/core/Engine.js:23]: Slack thread: https://consensys.slack.com/archives/C029D95J3K8/p1630702642076200?thread_ts=1630702129.075800&cid=C029D95J3K8

**Review by sethkfman — APPROVED** [review]
LGTM

**Review by cortisiko — APPROVED** [review]
looks good 🌮

**Pull request merged**